modules/gcs/main.tf

```diff
@@ -128,13 +128,43 @@ resource "google_storage_bucket_object" "objects" {
   }
 }
 
-resource "google_storage_bucket_iam_binding" "bindings" {
+resource "google_storage_bucket_iam_binding" "authoritative" {
```

Comment:
Can you add a moved block for this?

Reply:
no, I can't because there is a new resource binding called exactly the same